doc_source/distribution-web-values-specify.md

```diff
@@ -695,4 +695,4 @@ The countries that you want to add to your whitelist or blacklist\. To add a cou
 + To add multiple consecutive countries, select the first country, press and hold the Shift key, select the last country, and choose **Add**\. 
 + To add multiple non\-consecutive countries, select the first country, press and hold the Ctrl key, select the remaining countries, and choose **Add**\.
 + To find a country in the left list, enter the first few characters of the country's full name\. 
-+ The two\-letter code before the name of each country is the value that you enter if you want to create or update a distribution by using the CloudFront API\. We use the International Organization for Standardization country codes\. For an easy\-to\-use list, sortable by code and by country name, see the Wikipedia entry [ISO 3166\-1 alpha\-2](http://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)\.
\ No newline at end of file
++ The two\-letter code before the name of each country is the value that you enter if you want to create or update a distribution by using the CloudFront API\. We use the International Organization for Standardization country codes\. For an easy\-to\-use list, sortable by code and by country name, see the Wikipedia entry [ISO 3166\-1 alpha\-2](http://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)\.
```

Comment:
github editor changed newline at end of file

Reply:
Thanks! :-)